keyboards/rura66/rev1/config.h

```diff
@@ -0,0 +1,160 @@
+/*
+Copyright 2021 T.Shinohara
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x04D8
+#define PRODUCT_ID      0xE8DB
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    ShinoharaTa
+#define PRODUCT         RURA66
+
+/* key matrix size */
+#define MATRIX_ROWS 10
+#define MATRIX_COLS 7
+
+/*
+ * Keyboard Matrix Assignments
+ *
+ * Change this to how you wired your keyboard
+ * COLS: AVR pins used for columns, left to right
+ * ROWS: AVR pins used for rows, top to bottom
+ * DIODE_DIRECTION: COL2ROW = COL = Anode (+), ROW = Cathode (-, marked on diode)
+ *                  ROW2COL = ROW = Anode (+), COL = Cathode (-, marked on diode)
+ *
+ */
+#define MATRIX_ROW_PINS { D4, C6, D7, E6, B4 }
+#define MATRIX_COL_PINS { F4, F5, F6, F7, B1, B3, B2 }
+#define UNUSED_PINS
+
+/* COL2ROW, ROW2COL*/
+#define DIODE_DIRECTION COL2ROW
+
+/* Split hand configration */
+#define SPLIT_HAND_MATRIX_GRID D7,B2
+#define SPLIT_HAND_MATRIX_GRID_LOW_IS_RIGHT
+
```

Comment:
Just a heads up, you probably want to use the matrix mask option too, or this may keep the keyboard awake.

Reply:
Not sure how to do this. I have done some research on how to do it but came up empty. The only thing I could find on the subject was this pull request. https://github.com/qmk/qmk_firmware/issues/845 and this information:

Note that adding a diode at a previously unused intersection will effectively tell the firmware that there is a key held down at that point. You can instruct qmk to ignore that intersection by defining MATRIX_MASKED and then defining a matrix_row_t matrix_mask[MATRIX_ROWS] array in your keyboard config. Each bit of a single value (starting form the least-significant bit) is used to tell qmk whether or not to pay attention to key presses at that intersection.
While MATRIX_MASKED isn't necessary to use SPLIT_HAND_MATRIX_GRID successfully, without it you may experience issues trying to suspend your computer with your keyboard attached as the matrix will always report at least one key-press.

I couldn't find a keyboard with a matrix mask to reference. Any help is appreciated. Thanks.

Reply:
It's mentioned here: 
https://docs.qmk.fm/#/feature_split_keyboard?id=handedness-by-matrix-pin

And the pisces keyboard uses it, if you want an example of this.